lms/djangoapps/grades/tasks.py

```diff
@@ -115,6 +115,25 @@ def recalculate_subsection_grade_v3(self, **kwargs):
     _recalculate_subsection_grade(self, **kwargs)
 
 
+import time
+from celery import group
+@task()
+def add(x, y):
+    time.sleep(4)
+    return x + y
+
+
+@task
+def tsum(x):
+    time.sleep(8)
+    return sum(x)
+
+
+@task
+def do_stuff():
+    thing = group(add.s(i, i) for i in xrange(10))
+    thing.apply_async()
```

Comment:
Just for funsies, here's what chunking looks like (line 134 updated to `thing = add.chunks(zip(range(10), range(10)), 2)`, same console commands):

```
2017-04-20 00:57:35,532 WARNING 9145 [celery.redirected] log.py:282 - celery@vagrant ready.
2017-04-20 00:58:02,371 INFO 9145 [celery.worker.strategy] strategy.py:55 - Received task: lms.djangoapps.grades.tasks.do_stuff[f89ffd86-dfb4-4d2b-a303-d4084c93bed0]
2017-04-20 00:58:02,419 INFO 9145 [celery.worker.strategy] strategy.py:55 - Received task: celery.starmap[68f12142-1798-42f9-92cd-e0386c433811]
2017-04-20 00:58:02,421 INFO 9145 [celery.worker.strategy] strategy.py:55 - Received task: celery.starmap[106088c6-dd0c-4d15-9a45-2373bb925790]
2017-04-20 00:58:02,422 INFO 9145 [celery.worker.job] job.py:401 - Task lms.djangoapps.grades.tasks.do_stuff[f89ffd86-dfb4-4d2b-a303-d4084c93bed0] succeeded in 0.0491187229927s: None
2017-04-20 00:58:02,425 INFO 9145 [celery.worker.strategy] strategy.py:55 - Received task: celery.starmap[b54eeb07-35ef-49f9-83c1-b727ec9b5410]
2017-04-20 00:58:02,428 INFO 9145 [celery.worker.strategy] strategy.py:55 - Received task: celery.starmap[7292b3c7-d541-4e7b-b5fb-29a9b343481a]
2017-04-20 00:58:10,436 INFO 9145 [celery.worker.job] job.py:401 - Task celery.starmap[68f12142-1798-42f9-92cd-e0386c433811] succeeded in 8.012498072s: [0, 2]
2017-04-20 00:58:10,437 INFO 9145 [celery.worker.strategy] strategy.py:55 - Received task: celery.starmap[89f404b2-462d-4aff-b863-a37147381a1b]
2017-04-20 00:58:10,439 INFO 9145 [celery.worker.job] job.py:401 - Task celery.starmap[106088c6-dd0c-4d15-9a45-2373bb925790] succeeded in 8.016452879s: [4, 6]
2017-04-20 00:58:18,445 INFO 9145 [celery.worker.job] job.py:401 - Task celery.starmap[7292b3c7-d541-4e7b-b5fb-29a9b343481a] succeeded in 8.005027484s: [12, 14]
2017-04-20 00:58:18,446 INFO 9145 [celery.worker.job] job.py:401 - Task celery.starmap[b54eeb07-35ef-49f9-83c1-b727ec9b5410] succeeded in 8.00928362799s: [8, 10]
2017-04-20 00:58:26,453 INFO 9145 [celery.worker.job] job.py:401 - Task celery.starmap[89f404b2-462d-4aff-b863-a37147381a1b] succeeded in 8.00566811801s: [16, 18]
```